projectile.el

```diff
@@ -475,7 +475,10 @@ The saved data can be restored with `projectile-unserialize'."
 
 (defvar projectile-known-projects nil
   "List of locations where we have previously seen projects.
-The list of projects is ordered by the time they have been accessed.")
+The list of projects is ordered by the time they have been accessed.
+
+See also `projectile-remove-known-project' to interactively
```

Comment:
Aha, I didn't know those existed either! `projectile-cleanup-known-projects` is what I want most of the time, so this PR has been worthwhile just to learn about that :)